src/controllers/FacetQueryController.ts

```diff
@@ -311,6 +310,9 @@ export class FacetQueryController {
   }
 
   private processQueryOverrideForAdditionalFilter(queryBuilder: QueryBuilder, mergeWith: QueryBuilderExpression) {
```

Comment:
If we are adding the same logic when the additionalFilter is specified, will the counts be correct when using this option?

Reply:
Yes because it also will send the other overrides, the advanced query for example will contain the filter from the other facets